Multi-line aggregation did not explain date/time quirks (#25876)
DataDog/documentation: content/en/agent/logs/advanced_log_collection.md

{{% /tab %}}
{{< /tabs >}}

> <div class="alert alert-warning"><strong>Important!</strong> Regex patterns for multi-line logs must start at the <em>beginning</em> of a log. Patterns cannot be matched mid-line. <em>A never matching pattern may cause log line losses.</em> Also keep in mind that log collection works with precision of up to millisecond, so logs with greater precision are not sent even if they match the pattern.</div>

Reviewer: ```suggestion
<div class="alert alert-warning"><strong>Important!</strong> Regex patterns for multi-line logs must start at the <em>beginning</em> of a log. Patterns cannot be matched mid-line. <em>A never matching pattern may cause log line losses.</em> <br><br>Log collection works with precision of up to millisecond. Logs with greater precision are not sent even if they match the pattern.</div>
```

> <div class="alert alert-warning"><strong>Important!</strong> Regex patterns for multi-line logs must start at the <em>beginning</em> of a log. Patterns cannot be matched mid-line. <em>A never matching pattern may cause log line losses.</em> Also keep in mind that log collection works with procesion of up to millisecond, so logs with greater precision will not be sent even if they match the pattern.</div>

Reviewer 2: ```suggestion
<div class="alert alert-warning"><strong>Important!</strong> Regex patterns for multi-line logs must start at the <em>beginning</em> of a log. Patterns cannot be matched mid-line. <em>A never matching pattern may cause log line losses.</em> Also keep in mind that log collection works with precision of up to millisecond, so logs with greater precision are not sent even if they match the pattern.</div>
```

did you mean "procession" or "precision" here?

Author: Yes, that was a typo, thank you
